newsfragments/41420.significant.rst

```diff
@@ -0,0 +1 @@
+Replaced Python's `list` with `collections.abc.MutableSet` for the `Dag` tags.
```

Comment:
You might include guidance on how the user can adapt to the change (for example when discussing the cluster policy.)

Reply:
Where do I add it? To the same file?

Reply:
Yes

Reply:
I think I have fixed it, what do you think?